client/utils/build-challenges.ts

```diff
@@ -1,19 +1,24 @@
-const _ = require('lodash');
+/* eslint-disable @typescript-eslint/no-unsafe-call */
+/* eslint-disable @typescript-eslint/no-unsafe-return */
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
 
-const {
+import _ from 'lodash';
+
+import {
   getChallengesForLang,
   createChallenge,
   challengesDir,
   getChallengesDirForLang
-} = require('../../curriculum/getChallenges');
-const envData = require('../../config/env.json');
```

Comment:
@ojeytonwilliams Was there not a reason this was being required instead of imported?

Reply:
Sorry, I’ll fix that

Reply:
@samarmohan There may not be a need to fix this. I am getting clarification from Oliver, whether or not this change could break things.

Reply:
No reason I can think of. It won't fail quietly, so if the build works at all with this change it should be fine.